client/src/components/AdvisorReports/AdvisorReportsRow.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import Checkbox from 'components/Checkbox'
+import _uniqueId from 'lodash.uniqueid'
+
+const _advisorStats = (columnGroups, statistics) => {
+    let stats = []
+    columnGroups.forEach( (group, index) => {
+        let rowCell = statistics[index]
+        let keySubmitted = _uniqueId('submitted_')
+        let keyAttended = _uniqueId('attended_')
+        if(rowCell){
+            stats.push(<td key={ keySubmitted }>{ rowCell.nrreportssubmitted }</td>)
+            stats.push(<td key={ keyAttended }>{ rowCell.nrengagementsattended }</td>)
+        }
+        else {
+            stats.push(<td key={ keySubmitted }>0</td>)
+            stats.push(<td key={ keyAttended }>0</td>)
+        }
+    })
+    return stats
+}
+
+const AdvisorReportsRow = (props) => {
+    let statistics = _advisorStats(props.columnGroups, props.row.stats)
```

Comment:
I would add here the propTypes. That way you force certain property types and you also document the component better.